double_entry.gemspec

```diff
@@ -62,6 +62,7 @@ POSTINSTALLMESSAGE
   gem.add_development_dependency 'generator_spec'
   gem.add_development_dependency 'machinist'
   gem.add_development_dependency 'timecop'
+  gem.add_development_dependency 'test-unit'
```

Comment:
Ruby 2.2 doesn't supply `test-unit` while the previous versions do. Rails 3.2 requires this.